client/reader/stream/index.jsx

```diff
@@ -92,6 +92,10 @@ function sameSite( postKey1, postKey2 ) {
 		);
 }
 
+function sameDay( postKey1, postKey2 ) {
+	return postKey1.localMoment.date() === postKey2.localMoment.date(); // date === day of the month
```

Comment:
might want to include the month here too. given how we use it, collisions are unlikely, but better safe than weird.